WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersPeriodViewController.swift

```diff
@@ -142,7 +142,7 @@ final class StoreStatsAndTopPerformersPeriodViewController: UIViewController {
     }
 
     deinit {
-        cancellables.forEach {
+        subscriptions.forEach {
             $0.cancel()
         }
```

Comment:
I'm not sure if we really need this because Combine's subscriptions should be cleaned up along with the controller they belong to, right?